src/hypothesis/extra/datetime.py

```diff
@@ -15,70 +15,194 @@
 #
 # END HEADER
 
+"""As might be expected, this provides strategies for which generating
```

Comment:
We don't need "As might be expected" – it doesn’t add anything and just annoys the reader who might be expecting something different.